contracts/interfaces/access-control/IMultiOwnable.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.4;
+
+interface IMultiOwnable {  
+
+    event OwnershipAdded(address[] indexed newOwners);
+    event OwnershipRemoved(address[] indexed removedOwners); 
```

Comment:
Actually, broadly speaking, you can't add or remove "ownership". Let's name these 

`OwnerAdded` and `OwnerRemoved`  respectively